rskj-core/src/main/java/org/ethereum/sync/SyncPool.java

```diff
@@ -75,39 +76,39 @@
     private final SystemProperties config;
     private final NodeManager nodeManager;
     private final PeerClientFactory peerClientFactory;
+    private final ScheduledExecutorService syncPoolExecutor;
 
     public SyncPool(EthereumListener ethereumListener, Blockchain blockchain, SystemProperties config, NodeManager nodeManager, PeerClientFactory peerClientFactory) {
         this.ethereumListener = ethereumListener;
         this.blockchain = blockchain;
         this.config = config;
         this.nodeManager = nodeManager;
         this.peerClientFactory = peerClientFactory;
-
-        if (!config.isSyncEnabled()) {
-            return;
-        }
-
+        this.syncPoolExecutor = Executors.newSingleThreadScheduledExecutor(target -> new Thread(target, "syncPool"));
         updateLowerUsefulDifficulty();
+    }
 
-        Executors.newSingleThreadScheduledExecutor().scheduleWithFixedDelay(
-                new Runnable() {
-                    @Override
-                    public void run() {
-                        try {
-                            heartBeat();
-                            processConnections();
-                            updateLowerUsefulDifficulty();
-                            fillUp();
-                            prepareActive();
-                        } catch (Throwable t) {
-                            logger.error("Unhandled exception", t);
-                            panicProcessor.panic("syncpool", String.format("Unhandled exception {}", t.getMessage()));
-                        }
-                    }
-                }, WORKER_TIMEOUT, WORKER_TIMEOUT, TimeUnit.SECONDS
+    public void start() {
+        syncPoolExecutor.scheduleWithFixedDelay(
+            () -> {
+                try {
+                    heartBeat();
+                    processConnections();
+                    updateLowerUsefulDifficulty();
+                    fillUp();
+                    prepareActive();
+                } catch (Throwable t) {
+                    logger.error("Unhandled exception", t);
+                    panicProcessor.panic("syncpool", String.format("Unhandled exception {}", t.getMessage()));
```

Comment:
Can you fix this log? I think it should be `panicProcessor.panic("syncpool", String.format("Unhandled exception %s", t.getMessage()));`